src/squawk/SquawkAssignment.h

```diff
@@ -87,6 +87,10 @@ namespace UKControllerPlugin {
                     const UKControllerPlugin::Euroscope::EuroScopeCFlightPlanInterface & flightPlan,
                     const UKControllerPlugin::Euroscope::EuroScopeCRadarTargetInterface & radarTarget
                 ) const;
+                bool NeedsSquawkFlightPlan(

```

Comment:
Apologies for the delay in re-review, didn't get the notification!

The build is currently failing on Appveyor - I think because of Line 91. The type names need to be fully qualified (e.g. `UKControllerPlugin::Euroscope::EuroScopeCFlightPlanInterface`).

Would you be able to add to the test suite for this function - testing it through one of the public methods would be enough. Let me know if this is something you need help with and I'd be happy to :)